new: comparision between osm wiki page creation and first use in data

tyrasd opened the pull request.

tyrasd committed:
add new viz: comparision between osm wiki page creation and first use in data

mocnik-science commented:
Thank you very much for providing this visualization! I will merge the commit into the `develop` branch to perform some code cleanup and some styling before merging into the `master` branch.

mocnik-science reviewed (changes requested):
Thank you very much for providing this visualization! How did you create the data.csv file? It would be great if there would be a way to create it automatically. The idea is that some script located in `data-mining` is used to create a data set, which is then saved in `data`. These data sets should be accessible to everyone and thus be universal.

If you want me to assist in writing the (automatic) data mining part, I can contribute!

I will even do some code cleanup, in particular:
* use sass
* remove unnecessary style and script imports
* remove commented code blocks
* use single quotation marks

Otherwise, the code looks very clean and well-formatted!

tyrasd committed:
add data mining script for wiki-vs-osmdata viz, switch to json format

tyrasd committed:
drop unused libs, switch style to sass

tyrasd committed:
remove unused styles

tyrasd committed:
make sure that x=y background line gets transformed on zoom

tyrasd committed:
use single quoted strings

tyrasd committed:
add dummy help info

tyrasd commented:
I think I should have addressed all your points now.

Still missing:
* link/screenshor from overview page
* better help text / tooltips describing the UI (e.g. zooming by click-dragging)

tyrasd commented:
oh, one more thing: the data mining step requires the [`../../data/osm-tags-history-wiki.json`](https://github.com/mocnik-science/osm-vis/pull/1/files#diff-49c82cd9ca0e0c76645554d4a0f83a00R5) file to be present (i.e. the data mining step for the wiki visualization to be done before this one). I hope that's ok and doesn't add too much implicit internal dependency. :thinking:

mocnik-science commented:
Regarding the implicit dependencies of the data mining steps: that is fine, because it makes sense to data mine the wiki only once! I will have to write some script that performs the data mining steps automatically one after each other ... it does not matter then! You also described this in the `README.md`, so problem at all.

tyrasd committed:
drop accidentally commited file

mocnik-science commented:
nice!

mocnik-science commented:
visualizations/osm-tags-wiki-vs-osmdata introduced
